src/pg/brc20/brc20-pg-store.ts

```diff
@@ -111,38 +125,38 @@ export class Brc20PgStore {
       FROM events
       INNER JOIN
     `;
+    // todo: use event history
```

Comment:
Can you explain what this would do?

Reply:
Not sure what exactly it would look like, but a table/history of events (for visualization) would be nice. I noticed the method already existed but wasn't used.